downstream/assemblies/aap-on-azure/assembly-aap-azure-hub-spoke-peering.adoc

```diff
@@ -0,0 +1,33 @@
+ifdef::context[:parent-context: {context}]
+
+[id="assembly-aap-azure-hub-spoke-peering"]
+= Hub-and-spoke peering (Transit routes)
+
+:context: aap-azure-hub-spoke-peering
+
+// [role="_abstract"]
+// You can use these instructions to install 
+
+[NOTE]
+====
+Updating route tables incorrectly can break your network. Only execute the steps in these procedures if you are confident that you can reverse any unexpected network behavior.
+====
+
+== Prerequisites
+
+* You have deployed Ansible Automation Platform on Microsoft Azure
+* You have configured and tested an Azure VNET hub-and-spoke implementation in your Azure tenant
+* You have configured transit routing between your spoke networks, including your VPNs.
+* You have identified the following:
+** The CIDR block(s) of your existing VNETs (including VPNs & direct connects) that will need access to Ansible Automation Platform on Azure UIs
+** The CIDR block(s) of your existing VNETs (including VPNs & direct connects) that will contain hosts or endpoints for Ansible automation
+** The CIDR blocks of the Ansible Automation Platform Kubernetes network from the managed resource group of the application. Refer to Finding the CIDR block of the managed application kubernetes cluster for instructions.
```

Comment:
> Refer to Finding the CIDR block of the managed application kubernetes cluster for instructions.

Should there be a link here?

Reply:
Yes, there should be, thank you!

Reply:
Suggestion: For bulleted and nested lists, add a period at the end of each sentence/fragment. IBM Style Guide.